### ISSUE TITLE
Video stuttering/unresponsive following pause and resume

### ISSUE THREAD

**Issue by mjnewton** [nova-video-player/aos-AVP#466]
Nova Version: 5.15.20-20210509.1910
Device: Amazon 4K Firestick with wired 100Mbps Ethernet adapter
TV: Sony XH9505 (2020)
Video Source: Samba Fileserver (on Ubuntu 20.04) with Gigabit Ethernet adapter

Issues/Symptoms:

- If I press pause then on resuming playback the video plays very slowly and the user interface becomes unresponsive
- If I fast-forward/rewind the video sometimes freezes on release

Repeatability:

- I've not pinned it down to specific videos, formats etc but it's happening a lot
- Once it's started it doesn't seem to ever recover without a restart

Workaround:

- Having disabled SMBv2 (and enabled SMBv1 on the Samba server) the problem appears to go away (reenabling SMBv2 makes i t return)

More than happy to some further digging/diagnosis to help pinpoint or highlight what the issue might be.

**Comment by mjnewton** [comment]
Looks like I spoke too soon - I've just had the same issue with SMBv1 too.

So, it looks like both my title and subsequent diagnosis are incorrect. The thing is, I'm not sure what else to say now as I've got nothing to pin it down further!

**Comment by mjnewton** [comment]
I've attached a log below. I paused and resumed a few times without issue before triggering the problem at (I think) 08-31 20:23:29.220:

08-31 20:22:54.135 14675 15352 V AvosMediaPlayer: Initializing AvosMediaPlayer
08-31 20:22:54.156 14675 15356 V avos_media_player: event_thread: start
08-31 20:22:54.158 14675 14701 D avos_player: 0x8341ec00|avos_mp_create:
08-31 20:22:54.185 14675 14701 D avos_player: 0x8341ec00|avos_mp_setdatasource: http://localhost:40097/video.mp4
08-31 20:22:54.186 14675 15352 D avos_media_player: setVideoSurface
08-31 20:22:54.186 14675 15352 V avos_media_player: free_native_window
08-31 20:22:54.188 14675 14701 D avos_player: 0x8341ec00|avos_mp_open_async:
08-31 20:22:54.189 14675 14701 D avos_player:
08-31 20:22:54.189 14675 14701 D avos_player: stream_open http://localhost:40097/video.mp4  etype 15->ETYPE_MP4  flags 2001
08-31 20:22:54.189 14675 14701 D avos_player: flags:  PAUSED  LATE_INDEX  FILE_NONLOCAL
08-31 20:22:55.219 14675 14701 D avos_player: PARSER: [FFMPEG]  buffersize -1
08-31 20:22:55.219 14675 14701 D avos_player: VIDEO:  [H264] [H.264] 1920x800   0fps  2249kbit/s  dec [(none)]
08-31 20:22:55.219 14675 14701 D avos_player: AUDIO:  [00FF] [AAC] 48000Hz  6-ch 0bit  384kbit/s  dec [(none)]
08-31 20:22:55.219 14675 14701 D avos_player: audio[0] playable!
08-31 20:22:55.224 14675 15369 I OMXClient: MuxOMX ctor
08-31 20:22:55.250 14675 15368 I MediaCodec: [OMX.MTK.VIDEO.DECODER.AVC] setting surface generation to 15027201
08-31 20:22:55.250 14675 15368 I MediaCodecLogger: [ASAP] HW.video.avc P2 + 7594386
08-31 20:22:55.270 14675 15368 I MediaCodecLogger: [ASAP] HW.video.avc P2 - 7594406
08-31 20:22:55.271 14675 15368 D MediaCodecLogger: Got a Service. Calling PID = 14675
08-31 20:22:55.272 14675 15355 I MediaCodecLogger: [ASAP] HW.video.avc P3 + 7594408
08-31 20:22:55.294 14675 15369 D SurfaceUtils: set up nativeWindow 0x801b3008 for 1920x800, color 0x7f000001, rotation 0, usage 0x2933
08-31 20:22:55.294 14675 15369 W ACodec  : [OMX.MTK.VIDEO.DECODER.AVC] setting nBufferCountActual to 7 failed: -2147483648
08-31 20:22:55.301 14675 15368 I MediaCodecLogger: [ASAP] HW.video.avc P3 - 7594436
08-31 20:22:55.301 14675 14701 D avos_player: VID_SNK: [sfdec2]
08-31 20:22:55.301 14675 14701 D avos_player: videosink_open: opened
08-31 20:22:55.301 14675 14701 D avos_player: VID_DEC: [sfdec2]
08-31 20:22:55.303 14675 14701 D avos_player: AUD_DEC: [ffmpeg]
08-31 20:22:55.303 14675 14701 D avos_player: facomp: open
08-31 20:22:55.303 14675 14701 D avos_player: lvl 0  tgt 0  maxg 0  sm 8  hist 65536
08-31 20:22:55.303 14675 14701 D avos_player: facomp: level 0
08-31 20:22:55.303 14675 14701 D avos_player:
08-31 20:22:55.303 14675 14701 D avos_player: stream_start http://localhost:40097/video.mp4
08-31 20:22:55.303 14675 14701 D avos_player: AUD_SNK: [audio]
08-31 20:22:55.303 14675 14701 D avos_player: stream_sink_audio_open
08-31 20:22:55.303 14675 14675 W Thread-23: type=1400 audit(0.0:203): avc: denied { read } for name="u:object_r:dolby_prop:s0" dev="tmpfs" ino=1307 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:dolby_prop:s0 tclass=file permissive=0
08-31 20:22:55.313 14675 15355 E libc    : Access denied finding property "dolby.dma.minbufcnt"
08-31 20:22:55.313 14675 15355 D AudioTrack: min buf count is limited to 2 times HAL buf
08-31 20:22:55.325 14675 15355 D AmazonAudioTrackCallback: No AudioTrackFlags provided.  Using internal defaults.
08-31 20:22:55.330 14675 15355 D AmazonAudioTrackCallback: No Audio content type provided.
08-31 20:22:55.330 14675 15355 I android.media.AudioTrack: AUDIOINFO: audio_input: format: 2 sample_rate: 48000 channels: 6 AudioAttributes: AudioAttributes: usage=1 content=2 flags=0x0 tags= bundle=null
08-31 20:22:55.333 14675 14675 W Thread-23: type=1400 audit(0.0:204): avc: denied { read } for name="u:object_r:dolby_prop:s0" dev="tmpfs" ino=1307 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:dolby_prop:s0 tclass=file permissive=0
08-31 20:22:55.339 14675 15355 E libc    : Access denied finding property "dolby.dma.minbufcnt"
08-31 20:22:55.339 14675 15355 D AudioTrack: min buf count is limited to 2 times HAL buf
08-31 20:22:55.344 14675 15355 D AudioTrack: Client defaulted notificationFrames to 1025 for frameCount 3076
08-31 20:22:55.367 14675 14701 D avos_player: codec not supported for passthrough...
08-31 20:22:55.367 14675 14701 D avos_player: start video at start
08-31 20:22:55.363 14675 14675 W Thread-24: type=1400 audit(0.0:205): avc: denied { read } for name="u:object_r:dolby_prop:s0" dev="tmpfs" ino=1307 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:dolby_prop:s0 tclass=file permissive=0
08-31 20:22:55.368 14675 15390 E libc    : Access denied finding property "dolby.dma.minbufcnt"
08-31 20:22:55.368 14675 15390 D AudioTrack: min buf count is limited to 2 times HAL buf
08-31 20:22:55.372 14675 15390 D AmazonAudioTrackCallback: No AudioTrackFlags provided.  Using internal defaults.
08-31 20:22:55.374 14675 15390 D AmazonAudioTrackCallback: No Audio content type provided.
08-31 20:22:55.374 14675 15390 I android.media.AudioTrack: AUDIOINFO: audio_input: format: 2 sample_rate: 48000 channels: 6 AudioAttributes: AudioAttributes: usage=1 content=2 flags=0x0 tags= bundle=null
08-31 20:22:55.388 14675 15390 E libc    : Access denied finding property "dolby.dma.minbufcnt"
08-31 20:22:55.383 14675 14675 W Thread-24: type=1400 audit(0.0:206): avc: denied { read } for name="u:object_r:dolby_prop:s0" dev="tmpfs" ino=1307 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:dolby_prop:s0 tclass=file permissive=0
08-31 20:22:55.389 14675 15390 D AudioTrack: min buf count is limited to 2 times HAL buf
08-31 20:22:55.391 14675 15390 D AudioTrack: Client defaulted notificationFrames to 1025 for frameCount 3076
08-31 20:22:55.400 14675 14675 I com.archos.mediacenter.video.player.PlayerActivity: [main] INFO  c.a.m.video.player.PlayerActivity - onAudioMetadataUpdated: newAudio: 0  mVideoInfo.audioTrack: 0  firstTimeUpdated: true  nbTrack: 1
08-31 20:22:55.402 14675 14675 W ResourceType: No package identifier when getting value for resource number 0x00000000
08-31 20:22:55.403 14675 14675 W ResourceType: No package identifier when getting value for resource number 0x00000000
08-31 20:22:55.758 14675 14701 D avos_player: 0x8341ec00|avos_mp_setaudiofilter: 0 0
08-31 20:22:55.758 14675 14701 D avos_player: facomp: level 0
08-31 20:22:55.852 14675 14675 I com.archos.mediacenter.video.player.PlayerActivity: [main] INFO  c.a.m.video.player.PlayerActivity - onAudioMetadataUpdated: newAudio: -1  mVideoInfo.audioTrack: 0  firstTimeUpdated: false  nbTrack: 1
08-31 20:22:55.853 14675 14675 W ResourceType: No package identifier when getting value for resource number 0x00000000
08-31 20:22:55.854 14675 14675 W ResourceType: No package identifier when getting value for resource number 0x00000000
08-31 20:22:55.858 14675 14701 D avos_player: 0x8341ec00|avos_mp_setavdelay: 0
08-31 20:22:55.895 14675 14701 D avos_player: stream_play_n_frames( 10, 3498291, -1 )
08-31 20:22:55.904 14675 15368 I MediaCodecLogger: [ASAP] HW.video.avc Got First Input Frame 7595039
08-31 20:22:55.948 14675 15369 W AHierarchicalStateMachine: Warning message AMessage(what = 'omxI') = {
08-31 20:22:55.948 14675 15369 W AHierarchicalStateMachine:   int32_t type = 0
08-31 20:22:55.948 14675 15369 W AHierarchicalStateMachine:   int32_t event = 3
08-31 20:22:55.948 14675 15369 W AHierarchicalStateMachine:   int32_t data1 = 1
08-31 20:22:55.948 14675 15369 W AHierarchicalStateMachine:   int32_t data2 = 117440527
08-31 20:22:55.948 14675 15369 W AHierarchicalStateMachine: } unhandled in root state.
08-31 20:22:55.954 14675 15369 D SurfaceUtils: set up nativeWindow 0x801b3008 for 1920x1088, color 0x7f000001, rotation 0, usage 0x2933
08-31 20:22:55.955 14675 15369 W ACodec  : [OMX.MTK.VIDEO.DECODER.AVC] setting nBufferCountActual to 14 failed: -2147483648
08-31 20:22:55.988 14675 15368 I MediaCodecLogger: updateFormatChanged width = 1920 height = 800
08-31 20:22:55.988 14675 14701 D avos_player: init_renderer: width NOT changed: 1920 -> 1920
08-31 20:22:55.988 14675 15368 I MediaCodecLogger: updatePTSTime [HW.video.avc] First PTS after Flush or reset = 3498291000
08-31 20:22:55.988 14675 15368 I MediaCodecLogger: [ASAP] HW.video.avc Got First Frame Ready 7595124
08-31 20:22:55.990 14675 15368 I MediaCodecLogger: [ASAP] HW.video.avc Got First Frame Render 7595125
08-31 20:22:55.990 14675 15368 I MediaCodecLogger: App uses Timestamps for AVSync
08-31 20:22:55.990 14675 14701 D avos_player: init_renderer: height NOT changed: 800 -> 1088
08-31 20:22:55.990 14675 14701 D avos_player: sfdec_read: INFO_FORMAT_CHANGED: 1920x800
08-31 20:22:56.000 14675 14701 D avos_player:   <NSR 3498291>0x8341ec00|avos_mp_start:
08-31 20:22:56.008 14675 14701 D avos_player: 0x8341ec00|avos_mp_setlooping: 0
08-31 20:22:56.008 14675 14701 D avos_player: 0x8341ec00|avos_mp_setaudiofilter: 0 0
08-31 20:22:56.008 14675 14701 D avos_player: facomp: level 0
08-31 20:22:56.563 14675 15390 W AudioTrack: dead IAudioTrack, PCM, creating a new one from obtainBuffer()
08-31 20:22:56.583 14675 14675 W Thread-24: type=1400 audit(0.0:207): avc: denied { read } for name="u:object_r:dolby_prop:s0" dev="tmpfs" ino=1307 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:dolby_prop:s0 tclass=file permissive=0
08-31 20:22:56.590 14675 15390 E libc    : Access denied finding property "dolby.dma.minbufcnt"
08-31 20:22:56.590 14675 15390 D AudioTrack: min buf count is limited to 2 times HAL buf
08-31 20:22:56.598 14675 15390 D AudioTrack: Client defaulted notificationFrames to 1025 for frameCount 3076
08-31 20:22:57.057 14675 15390 W AudioTrack: dead IAudioTrack, PCM, creating a new one from obtainBuffer()
08-31 20:22:57.064 14675 15254 W AudioSystem: ioConfigChanged() closing unknown output 389
08-31 20:22:57.104 14675 15390 E libc    : Access denied finding property "dolby.dma.minbufcnt"
08-31 20:22:57.104 14675 15390 D AudioTrack: min buf count is limited to 2 times HAL buf
08-31 20:22:57.106 14675 15390 D AudioTrack: Client defaulted notificationFrames to 1025 for frameCount 3076
08-31 20:22:57.093 14675 14675 W Thread-24: type=1400 audit(0.0:208): avc: denied { read } for name="u:object_r:dolby_prop:s0" dev="tmpfs" ino=1307 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:dolby_prop:s0 tclass=file permissive=0
08-31 20:23:00.938 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 2054
08-31 20:23:01.020 14675 15368 I MediaCodecLogger: updatePTSTime Possible seek found!
08-31 20:23:01.020 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Resolution change from 0x0 to 1920x800 took = 20
08-31 20:23:03.142 14675 14701 D avos_player: #  <NSR 3498333>0x8341ec00|avos_mp_pause:
08-31 20:23:03.267 14675 14685 I art     : Background partial concurrent mark sweep GC freed 20910(1386KB) AllocSpace objects, 6(336KB) LOS objects, 5% free, 63MB/67MB, paused 1.664ms total 112.984ms
08-31 20:23:09.784 14675 14701 D avos_player: 0x8341ec00|avos_mp_start:
08-31 20:23:09.785 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 662
08-31 20:23:10.067 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:10.070 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:10.074 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:10.077 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:10.083 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:14.826 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 1780
08-31 20:23:16.080 14675 14701 D avos_player:   <NSR 3505541>0x8341ec00|avos_mp_pause:
08-31 20:23:18.516 14675 14701 D avos_player: 0x8341ec00|avos_mp_start:
08-31 20:23:18.803 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:18.805 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:18.807 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:18.810 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:19.874 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 666
08-31 20:23:21.158 14675 14701 D avos_player:   <NSR 3511958>0x8341ec00|avos_mp_pause:
08-31 20:23:23.461 14675 14701 D avos_player: 0x8341ec00|avos_mp_start:
08-31 20:23:24.921 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 960
08-31 20:23:25.198 14675 14701 D avos_player:   <NSR 3514666>0x8341ec00|avos_mp_pause:
08-31 20:23:25.661 14675 14701 D avos_player: 0x8341ec00|avos_mp_start:
08-31 20:23:25.675 14675 14695 I System.out: 20:23:25,672 |-INFO in ReconfigureOnChangeTask(born:1630437625670) - Empty watch file list. Disabling
08-31 20:23:25.947 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:25.948 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:25.950 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc Dropping frame
08-31 20:23:29.220 14675 14701 D avos_player:   <NSR 3516458>0x8341ec00|avos_mp_pause:
08-31 20:23:31.466 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 1122
08-31 20:23:31.472 14675 14701 D avos_player: 0x8341ec00|avos_mp_start:
08-31 20:23:36.556 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 428
08-31 20:23:41.781 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 723
08-31 20:23:47.005 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 773
08-31 20:23:52.230 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 1167
08-31 20:23:57.454 14675 15368 I MediaCodecLogger: com.android.mediaplayer.4K.HW.video.avc.bitrateInKbps = 893
08-31 20:23:57.989 14675 14685 I art     : Background partial concurrent mark sweep GC freed 52094(3MB) AllocSpace objects, 3(168KB) LOS objects, 5% free, 63MB/67MB, paused 1.199ms total 100.905ms

For what it's worth, I have another Firestick - **a non-4K version** - plugged into another TV and have never seen this issue on that one.

**Comment by mjnewton** [comment]
Is there anything I can do to help this be looked at?

**Comment by courville** [comment]
@mjnewton thanks for sharing the logs and analysis.
I will need to reproduce. Do you confirm that:
* you are not using passthrough mode (passthrough disabled) when using audiotrack advanced sound output
* it is kind of working with openSLES sound output option
* when you do not use your ethernet adapter (i.e. via wifi) are you able to reproduce the problem?
Could you please share a link to the movie sample to software at courville dot org?

**Comment by mjnewton** [comment]
Hi @courville, thanks for the input (and thanks for the software whilst I'm at it!)

I can confirm passthrough is *disabled* and, from what I can tell, the use of OpenSLES does appear to have resolved this.

I haven't tried using wifi though so will experiment with that. I have another firestick (non 4K, different TV) and I've never had this issue on that one... and that one happens to be connected via wifi so perhaps that is indeed relevant (or am I reading too much into your question on this point?).

I'll do some further digging, and will sort out a movie sample.

Incidentally, I'm pretty sure this issue I'm having is fairly recent (last few months?) and not something I'd seen previously, but that might be more related to a change at my end than yours (I'm thinking about the wired ethernet adapter in particular!).

**Comment by mjnewton** [comment]
Okay, after some further experimentation I can confirm I can recreate the issue over wifi too.

I've just sent a link to a video sample (and indeed a link to a different video that works totally fine).

As I've written in the email, the best (quickest) way to trigger the issue appears to be to continuously seek forwards then backwards, forwards then backwards - basically give it a really hard time! After 10-30s or so of doing this the blue circle bottom right spins and spins and the video either stops playing or proceeds really slowly (I think that's more after a pause/resume). Something I hadn't previously noticed is that whilst pressing forwards in this frozen state advances the blue bar to the right, so does pressing backwards too! Now, you might think 'well just dont fast forward and rewind so aggressively!' but I'm only using this method to trigger the symptom at will - it can still happen from a simple pause/resume but perhaps just less predictably.

**Comment by courville** [comment]
@mjnewton for some reasons I did not receive the link to the video.

**Comment by mjnewton** [comment]
Ah, okay, it looks like it got delivered but I wonder if Google's spam filter trapped it?

Sep 23 22:50:44 rugrat postfix/smtp[23841]: AC3B11C14DE: to=<software@courville.org>, relay=ASPMX.L.GOOGLE.COM[74.125.140.27]:25, delay=1.2, delays=0.01/0.03/0.32/0.88, dsn=2.0.0, status=sent (250 2.0.0 OK  1632433844 r6si8435126wma.35 - gsmtp)

Let me try again without a link and see if that's get through then I'll resend...

**Comment by courville** [comment]
@mjnewton thanks for sharing the sample clip.
I have tried to reproduce the issue on my side seeking forward/backward for several minutes without success so far.
I am using a firestick4k (first gen, not the max) over Wi-Fi 5GHz connected to a Synology NAS hosting the sample video file.
Will try more later but so far so good (I know it does not work on your side).

**Comment by courville** [comment]
One thing to check on your side is that when it happens, in the UI while playing the video (up key, info tile) if hardware decoding is reported (it could be that there is a codec error and then nova fallsback to software decoding).

**Comment by mjnewton** [comment]
Would it be of any use for my to video the issue occurring, if only to describe it better in pictures than what I can do in words?

I'll check the decoding type too, assuming the UI allows me too (I'm not sure if I've tried opening the on-screen menu).

**Comment by mjnewton** [comment]
> One thing to check on your side is that when it happens, in the UI while playing the video (up key, info tile) if hardware decoding is reported (it could be that there is a codec error and then nova fallsback to software decoding).

Only just had chance to test this and I confirm it *does* reporting Hardware Encoding when the issue occurs.

**Comment by Pentaphon** [comment]
> Would it be of any use for my to video the issue occurring, if only to describe it better in pictures than what I can do in words?

You should do that, yes, and can you post a link to the video (or a clip from the video) that is doing this? I have a 4K Stick as well so maybe I can be of help.

**Comment by courville** [comment]
@mjnewton, I would be interested to know if you see the issue only with mp4 container or does it happen with mkv containers too?

**Comment by mjnewton** [comment]
> @mjnewton, I would be interested to know if you see the issue only with mp4 container or does it happen with mkv containers too?

To be honest I need to revisit this and see if I've still got the issue now that I'm on v6.0.62. I updated and reset all my settings (I think), which would include the OpenSLES workaround, and I haven't heard anyone complaining about it freezing up on them.

If I can reproduce it still I'll certainly look if there's any pattern regarding mp4 vs mkv containers.

**Comment by mjnewton** [comment]
> @mjnewton, I would be interested to know if you see the issue only with mp4 container or does it happen with mkv containers too?

Apologies for the slow follow up on this.

Now that I'm on v6.0.66 (20220515.2014) and with audio interface set to Default Logic I can confirm the problem exists with both mp4 and mkv containers alike.

**Comment by mjnewton** [comment]
Finally got round to updating this with a video showing the locking up actually happening.

I am now on Nova v6.0.76 and the issue persists, and as shown in the video below it sometimes happens nearly straight away and other times takes a while. Seems to be quite random in terms of reproduction. I've ruled out the network/NAS completely as the video below was being played off the 4K Firestick's internal storage.

Video of it happening (twice): [https://www.newtonnet.co.uk/temp/novaissue466.mp4](https://www.newtonnet.co.uk/temp/novaissue466.mp4)
Test file that I am using: [https://www.newtonnet.co.uk/temp/novatest.mkv](https://www.newtonnet.co.uk/temp/novatest.mkv)

Can anyone (@courville ideally!) reproduce this and help determine what the issue is?
